CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ At the moment, this should always be done with the following `compat` admonition
 
 *By contributing code to Julia, you are agreeing to release it under the [MIT License](https://github.com/JuliaLang/julia/tree/master/LICENSE.md).*
 
-The Julia community uses [GitHub issues](https://github.com/JuliaLang/julia/issues) to track and discuss problems, feature requests, and pull requests (PR). You can make pull requests for incomplete features to get code review. The convention is to prefix the pull request title with "WIP:" for Work In Progress, or "RFC:" for Request for Comments when work is completed and ready for merging. This will prevent accidental merging of work that is in progress.
+The Julia community uses [GitHub issues](https://github.com/JuliaLang/julia/issues) to track and discuss problems, feature requests, and pull requests (PR).
+
+Issues and pull requests should have self explanatory titles such that they can be understood from the list of PRs and Issues.
+i.e. `Add feature x`, `Fix feature x`, `Add feature x (#12345) - fixup` are good, `Fixup #12345` is bad.
```

Comment:
```suggestion
i.e. `Add feature x` and `Fix issue x` are good, `Fix #12345. Corrects the bug.` is bad.
```
Typically they should not include numbers, since they do relatively poorly when trying to visually scan through a list of titles.